website/docs/configuration.md

```diff
@@ -239,6 +239,14 @@ Disable/enable Wave server boot during `wave run`. Defaults to `false`. Availabl
 
 The base URL (path prefix) to be used for resolving relative URLs (e.g. /foo/ or /foo/bar/, without the host (default "/"). If you run your Wave server (waved) and Wave app separately, it's necessary to set this env variable for both.
 
+### H2O_WAVE_NO_COPY_UPLOAD
+
+If your Wave server and Wave app run on the same machine, `q.site.upload` simply copies the files instead of pushing them through an HTTP request due to performance reasons (extra serialization). However, if you would like to force HTTP every time, you can se `H2O_WAVE_NO_COPY_UPLOAD` to `1` or `t` or `true`.
+
+### H2O_WAVE_WAVED_DIR
+
+Tell your Wave app the location of your Wave server if running on the same machine. Useful for performance optimizations during file uploads.
```

Comment:
```suggestion
Provides the location of the Wave server's root directory to the Wave app, if both the server and the app are running on the same machine. Useful for performance optimizations during file uploads. Makes `q.site.upload()` copy files instead of using HTTP requests.
```